tool/mongodb/start-server.sh

```diff
@@ -1,3 +1,3 @@
 set -eu
 
-sudo mongod --replSet rs0 --bind_ip localhost --config ./tool/mongodb/mongod.conf &
+sudo mongod --replSet rs0 --bind_ip localhost --config ./tool/mongodb/mongod.conf >&/dev/null &
```

Comment:
`mongod` keeps the SSH connection open by keeping the stdour/stderr open, so the script cannot progress unless we do this.